mappings/ace2ea0fe142a3687acf86f55bcded860a920864163ee0d3dda8b60252414b4552.json

```diff
@@ -43,7 +43,7 @@
     "policy": "820182018282051a05a6272c8200581c39a1df51147b6de6689a4727846962fb6540c3a3c7859a1a79b9420f",
     "logo": {
         "sequenceNumber": 0,
```

Comment:
Hi @kristopherfana,

you are updating logo, but not incrementing the sequenceNumber. Please refer to that guide and update the PR: https://github.com/cardano-foundation/cardano-token-registry/wiki/How-do-I-update-my-entry-in-the-registry%3F

Thanks!